src/util/goTo.js

```diff
@@ -0,0 +1,48 @@
+import { consoleError } from './console'
+
+/**
+ * Modified from https://github.com/alamcordeiro/vue-smooth-scroll
+ */
+
+function easeInOutCubic (t) { return t < .5 ? 4 * t * t * t : (t - 1) * (2 * t - 2) * (2 * t - 2) + 1 }
+
+export default function goTo (target, offset = 0) {
+  if (typeof window === 'undefined') return
+
+  if (offset && isNaN(offset)) {
+    consoleError(`Offset must be a Number, received ${offset.constructor.name} instead.`)
+    return
+  }
+
+  let end
+
+  if (typeof target === 'string') {
+    let anchor = target.charAt(0) === '#' ? target.substring(1) : target
+    let scrollTo = document.getElementById(anchor)
+    if(!scrollTo) return
```

Comment:
space after `if`, `yarn lint` should want you about it, if it didn't then it's something that should be fixed

Reply:
There's a few things in this PR that should be linting errors, but travis hasn't picked any of them up for some reason.